### DISCUSSION TITLE
Historic dates

### DISCUSSION THREAD

**Discussion by HemboPagi** [directus/directus#13616] [Archive]
Hello there

I am working with medieval dates and I have been wodering how to use date type without the calendar view. I would like to enter values manually as in most cases i have just a year or a year and a month. Any suggestions?

All the best
Hembo

**Comment by rijkvanzanten** [comment]
If you're just entering a year with an optional month, it sounds like you might want to use an `integer` with a regular numeric required input for the year, with an optional secondary input for the month, rather than a datetime interface 🙂

**Reply by HemboPagi** [reply]
True, but in some cases i have the full date, for example 1402.05.14.
So i can create 3 `integer` fields (which would be pain for the user) or one `string` filed. With last one then all the sorting and other date (number) range functionality would not work ...

**Reply by HemboPagi** [reply]
Played around with the data and still i feel that i need to use `date` type, but with the _input_ interface, where i could have date in predefine d format (for example 10.10.2020 or 10.2020 or 2020)
